applications/prompt-proto-service-lsstcomcam/values-usdfprod-prompt-processing.yaml

```diff
@@ -22,25 +22,37 @@ prompt-proto-service:
       # BLOCK-T60 is optics alignment
       # BLOCK-T75 is giant donuts
       # BLOCK-T88 is optics alignment
+      # BLOCK-T215 is evening twilight flats
+      # BLOCK-T216 is morning twilight flats
+      # BLOCK-T219 is first target test and should no longer be needed soon
```

Comment:
I don't know if I quite agree with this description -- the November 13 run seemed to be motivated by "we need more PR photos", so I think the block may have been co-opted as a generic "pretty pictures" script. But I can't find any documentation of that (or how/why it was run on the 13th).

Reply:
I was quite confused by some older slack messages that said they were going to remove the block in Oct but still used it last week. 

I modified the comment to be "pretty picture".